test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperParseTreeRewriterTest/AllowsPrefixedTagHelpers11.stree.txt

```diff
@@ -1,13 +1,34 @@
-Markup block - Gen<None> - 34 - (0:0,0)
-    Tag block - Gen<TagHelper> - 34 - (0:0,0) - th:myth2 - mythTagHelper2
-        SelfClosing - <th:myth2 bound="@DateTime.Now" />
-        bound - DoubleQuotes
-            Markup block - Gen<None> - 13 - (17:0,17)
-                Markup block - Gen<None> - 13 - (17:0,17)
-                    Expression block - Gen<Expr> - 13 - (17:0,17)
-                        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (17:0,17) - Tokens:1
-                            SyntaxKind.Transition;[@];
-                        Code span - Gen<Expr> - [DateTime.Now] - ImplicitExpressionEditHandler;Accepts:AnyExceptNewline;ImplicitExpression[ATD];K14 - (18:0,18) - Tokens:3
-                            SyntaxKind.Identifier;[DateTime];
-                            SyntaxKind.Dot;[.];
-                            SyntaxKind.Identifier;[Now];
+RazorDocument - [0..34)::34 - [<th:myth2 bound="@DateTime.Now" />]
+    MarkupBlock - [0..34)::34
+        MarkupTagHelperElement - [0..34)::34 - th:myth2[SelfClosing] - mythTagHelper2
+            MarkupTagHelperStartTag - [0..34)::34
+                MarkupTextLiteral - [0..9)::9 - [<th:myth2] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                    OpenAngle;[<];
+                    Text;[th:myth2];
+                MarkupTagHelperAttribute - [9..31)::22 - bound - DoubleQuotes
+                    MarkupTextLiteral - [9..10)::1 - [ ] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                        Whitespace;[ ];
+                    MarkupTextLiteral - [10..15)::5 - [bound] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                        Text;[bound];
+                    Equals;[=];
+                    MarkupTextLiteral - [16..17)::1 - ["] - Gen<None> - SpanEditHandler;Accepts:Any
+                        DoubleQuote;["];
+                    MarkupTagHelperAttributeValue - [17..30)::13
+                        MarkupDynamicAttributeValue - [17..30)::13 - [@DateTime.Now]
+                            GenericBlock - [17..30)::13
+                                CSharpCodeBlock - [17..30)::13
+                                    CSharpImplicitExpression - [17..30)::13
+                                        CSharpTransition - [17..18)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                                            Transition;[@];
+                                        CSharpImplicitExpressionBody - [18..30)::12
+                                            CSharpCodeBlock - [18..30)::12
+                                                CSharpExpressionLiteral - [18..30)::12 - [DateTime.Now] - Gen<Expr> - ImplicitExpressionEditHandler;Accepts:AnyExceptNewline;ImplicitExpression[ATD];K14
```

Comment:
This is an example of where we rewrite the `EditHandler` and `AcceptedCharacters` for a bound non-string attribute. Before the rewrite the AcceptedCharacters for this would be `NonWhitespace`.